packages/core/src/fhir-deduplication/__tests__/group-same-diag-reports.test.ts

```diff
@@ -49,11 +86,45 @@ describe("groupSameDiagnosticReports", () => {
     expect(diagReportsMap.size).toBe(2);
   });
 
+  it("discards diagReport if result and presented form are not present", () => {
+    diagReport.effectiveDateTime = dateTime.start;
+    diagReport.performer = [practRef];
+    delete diagReport.result;
+    delete diagReport.presentedForm;
+
+    const { diagReportsMap } = groupSameDiagnosticReports([diagReport]);
+    expect(diagReportsMap.size).toBe(0);
+  });
+
+  it("keeps diagReport if result is present", () => {
+    diagReport.effectiveDateTime = dateTime.start;
+    diagReport.performer = [practRef];
+
+    diagReport.result = resultExample;
+    delete diagReport.presentedForm;
```

Comment:
Should we make it clearer that the test is running with the expected data/state?
- "keeps diagReport if result is present" > "keeps diagReport if result is present and presentedForm isn't"
- add `expect(diagReport.result).toBeTruthy()`